### DISCUSSION TITLE
小白的本地部署教程的一些疑问

### DISCUSSION THREAD

**Discussion by CCodeInspect** [hhyo/Archery#2457] [General]
大佬能出一个本地部署的教程吗？

**Comment by CCodeInspect** [comment]
**有大佬说话吗**

**Comment by nick2wang** [comment]
参考wiki：https://github.com/hhyo/archery/wiki/manual

**Reply by CCodeInspect** [reply]
**大佬，初始化mysql的时候报无法创建外键。**
**报错信息：django.db.utils.IntegrityError: (1215, 'Cannot add foreign key constraint')**
另外：目前已创建的表如下：
![image](https://github.com/hhyo/Archery/assets/44918169/fd58b51c-7025-4902-b2f3-7a634314e985)

**Reply by CCodeInspect** [reply]
![image](https://github.com/hhyo/Archery/assets/44918169/bd02bc7a-8d19-480a-8052-d6b5299f44ee)
**大佬，这个次序需要怎么调整下比较好呢**

**Reply by CCodeInspect** [reply]
**已解决，只需要在DATABASES的OPTIONS的init_command中加入SET foreign_key_checks = 0就可以了。**
不能把OPTIONS下的init_command这句注释掉：
"init_command": "SET sql_mode='STRICT_TRANS_TABLES'",
